### DISCUSSION TITLE
Welcome to bot-github Discussions!

### DISCUSSION THREAD

**Discussion by AlbertAlonso** [AlbertAlonso/bot-github#5] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AlbertAlonso** [comment, marked as answer]
hi please commit files

**Reply by marioaldi** [reply]
yeah i cant

**Comment by AlbertAlonso** [comment]
how i youu

**Reply by AlbertAlonso** [reply]
yes

**Comment by marioaldi** [comment]
i amn fine

**Comment by AlbertAlonso** [comment]
adwasdawad

**Reply by marioaldi** [reply]
dwadsaw
